log.js

```diff
@@ -9,6 +9,15 @@ var ansi = require('ansi')
 log.cursor = ansi(process.stderr)
 log.stream = process.stderr
 
+// UTC timestamp, disabled by default.
+this.timestampEnabled = false
```

Comment:
In this context, `this` is going to be the module context, not, the log object. I think changing it to `log.timestampEnabled` will probably have the intended effect.